packages/server/lib/browsers/electron.js

```diff
@@ -161,6 +167,15 @@ module.exports = {
   _render (url, projectRoot, automation, options = {}) {
     const win = Windows.create(projectRoot, options)
 
+    if (options.browser.isHeadless) {
+      // seemingly the only way to force headless to a certain screen size
+      // electron BrowserWindow constructor is not respecting width/height options
+      win.setSize(options.width, options.height)
+    } else {
+      // we maximize in headed mode, this is consistent with chrome+firefox headed
+      win.maximize()
```

Comment:
We should only do this in run mode. In interactive mode, it's annoying to have electron maximize when it launches instead of being restored to its previous size and position.

Reply:
chrome launches maximized and I haven't heard many complaints about that

Reply:
Maybe the first time you open it? If you close Cypress-launched Chrome and then open it again, it restores itself to the last size and position in which you used it. We manually store the Electron size and position, so that we can do the same, but this was making it maximize every time instead of being restored to the last used size and position.